raft/raft.go

```diff
@@ -158,7 +158,10 @@ type raft struct {
 
 	// the leader id
 	lead uint64
-
+	// Leader transfer target.
+	transferee uint64
```

Comment:
// leadTransfree is id of the leader transfer target when its value is not zero.
leadTransfree?